packages/sku/context/defaultCompilePackages.js

```diff
@@ -9,9 +9,15 @@ let detectedCompilePackages = [];
 
 try {
   detectedCompilePackages = glob
-    .sync([`node_modules/@seek/*/package.json`], {
-      cwd: cwd(),
-    })
+    .sync(
+      [
+        'node_modules/@seek/*/package.json',
+        'node_modules/.pnpm/@seek*/node_modules/@seek/*/package.json',
```

Comment:
I don't think this will hold true when sku is run from a monorepo package, e.g. in Metropolis. The `.pnpm` directory exists only at the root of the monorepo.